project/Dependencies.scala

```diff
@@ -26,4 +26,9 @@ object Dependencies {
 
   val play = mainPlay ++ testPlay
 
+  def pickVersion(scalaVersion: String, ver211: String, ver212: String) = CrossVersion.partialVersion(scalaVersion) match {
```

Comment:
@romanowski ok, I reconsidered. I am pretty sure this think ( `CrossBuild` ) is used for something else. It basically is used for steering `Scala` version of given library not the library version.

So in essence:
`"com.mycompany" %% "lib" % "3.1.1"` is equal to 
`"com.mycompany" % "lib" % "3.1.1" cross CrossVersion.binary`

All the other variations of above (CrossVersion) are used to drive `XX` part of library: `com.mycompany; lib_XX; 3.1.1`